main.go

```diff
@@ -177,21 +201,9 @@ func main() {
 	if err != nil {
 		klog.Fatalf("net.ParseCIDR: %v", err)
 	}
-	if podIPNet.IP.To4() != nil {
-		klog.Fatalf("podCIDR is required to be IPv6 CIDR: %s", podCIDR)
-	}
 
-	podIPNetMaskSize, _ := podIPNet.Mask.Size()
-	// In stateless NAT algorithm, variable bits from pod CIDR are retained
-	// (for /120 pod CIDR, 1 byte is variable, for /112, 2 bytes, etc.).
-	// Those last bytes of IPv6 pod address are saved in IPv4 source address after IPv6 -> IPv4 NAT
-	// (for /120 pod CIDR, 198.18.0.0/16 IPv4 nat range and fd00:10:244:1::c5ff pod,
-	// bytes 0xc5 (197) and 0xff (255) will be saved, resulting in 198.18.197.255 source address).
-	// The wider IPv4 nat range is provided, the more pods we can handle without collissions, resulting
-	// in wider supported pod CIDR.
-	widestPodCIDRRangeAllowed := 128 - v4netMaskSize
-	if podIPNetMaskSize < widestPodCIDRRangeAllowed {
```

Comment:
we need 32 - v4netMaskSize  to obtain the number of available ips, so we can compare with 128 - podIPNetMaskSize